packages/account-abstraction/src/ERC4337EthersProvider.ts

```diff
@@ -170,7 +170,10 @@ export class ERC4337EthersProvider extends BaseProvider {
                 receipt: tx.receipt
               })}`
             )
+            tx.receipt.transactionHash = tx.receipt.hash
```

Comment:
This should be fixed in the relayer node or messaging server itself.. deleting field from response is not the right way to do this.

Reply:
WIP

Reply:
https://github.com/bcnmy/relayer-node-service/pull/81